docs/site/HasMany-relation.md

```diff
@@ -521,17 +521,145 @@ export class CustomerRepository extends DefaultCrudRepository {
   ];
   ```
 
-  Here is a diagram to make this more intuitive:
+{% include note.html content="The query syntax is a slightly different from LB3. We are also thinking about simplifying the query syntax. Check our GitHub issue for more information:
+[Simpler Syntax for Inclusion](https://github.com/strongloop/loopback-next/issues/3205)" %}
 
-  ![Graph](./imgs/hasMany-relation-graph.png)
+Here is a diagram to make this more intuitive:
+
+![Graph](./imgs/hasMany-relation-graph.png)
 
 - You can delete a relation from `inclusionResolvers` to disable the inclusion
   for a certain relation. e.g
   `customerRepository.inclusionResolvers.delete('orders')`
 
-{% include note.html content="
-Inclusion with custom scope:
-Besides specifying the relation name to include, it's also possible to specify additional scope constraints.
-However, this feature is not supported yet. Check our GitHub issue for more information:
-[Include related models with a custom scope](https://github.com/strongloop/loopback-next/issues/3453).
+### Query multiple relations
+
+It is possible to query several relations or nested include relations with
+custom scope. Once you have the inclusion resolver of each relation set up, the
+following queries would allow you traverse data differently:
+
+In our example, we have relations:
+
+- `Customer` _hasOne_ an `Address` - denoted as `address`.
+- `Customer` _hasMany_ `Order`s - denoted as `orders`.
+- `Order` _hasMany_ `Manufacturer` - denoted as `manufacturers`.
+
+To query **multiple relations**, for example, return all customers including
+their orders and address, in Node API:
+
+```ts
+customerRepo.find({include: [{relation: 'orders'}, {relation: 'address'}]});
+```
+
+Equivalently, with url, you can do:
+
+```
+GET http://localhost:3000/customers?filter[include][0][relation]=orders&filter[include][1][relation]=address
+```
+
+This gives
+
+```ts
+[
+  {
+    id: 1,
+    name: 'Thor',
+    addressId: 3
+    orders: [
+      {name: 'Mjolnir', customerId: 1},
+      {name: 'Rocket Raccoon', customerId: 1},
+    ],
+    address:{
+          id: 3
+          city: 'Thrudheim',
+          province: 'Asgard',
+          zipcode: '8200',
+    }
+  },
+  {
+    id: 2,
+    name: 'Captain',
+    orders: [{name: 'Shield', customerId: 2}], // doesn't have a related address
+  },
+]
+```
+
+To query **nested relations**, for example, return all customers including their
+orders and include orders' manufacturers , this can be done with filter:
+
+```ts
+customerRepo.find({
+  include: [
+    {
+      relation: 'orders',
+      scope: {
+        include: [{relation: 'manufacturers'}],
+      },
+    },
+  ],
+});
+```
+
+<!-- FIXME: the url isn't being converted to JSON correctly. Add an example url once it's fixed
```

Comment:
Maybe we can wait until https://github.com/strongloop/loopback-next/issues/2208 finished and see what's the generated url that explorer gives us, then update the example here.

Reply:
Sounds good